client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -303,6 +308,21 @@ class CalypsoifyIframe extends Component< Props & ConnectedProps & ProtectedForm
 			} );
 		}
 
+		if ( EditorActions.GetNavSidebarLabels === action ) {
+			const { translate } = this.props;
+
+			ports[ 0 ].postMessage( {
+				allPostsLabels: {
+					page: translate( 'View all pages' ),
+					post: translate( 'View all posts' ),
+				},
+				createPostLabels: {
+					page: translate( 'Create new page' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **99** times:
`translate( 'Create New Page' )` *ES Score: 7*

Reply:
Switching to sentence case